controllers/articles.controllers.js

```diff
@@ -6,13 +6,23 @@ exports.getArticles = (req, res) => {
 
         fetchArticles(req).then((data) => {
 
-            if(data.length === 0)
+           
+            
+             res.status(200).send({ data })
+        
+         }).catch((data) => {
```

Comment:
change data to err, it will always be an error that is caught here. you are just naming the parameter so it can be called anything but err makes more sense 😄